craft_parts/executor/executor.py

```diff
@@ -255,6 +257,7 @@ def _install_build_snaps(self) -> None:
                 ", ".join(build_snaps),
             )
         else:
+            logger.info("Installing requested build-snaps")
```

Comment:
Reading this feels a little better to me
```suggestion
        logger.info("Installing build-snaps")
```

Or perhaps required (as they were requested) :man_shrugging: